docs/fundamentals/dashboard/configuration.md

```diff
@@ -182,6 +182,7 @@ Telemetry limits have different scopes depending upon the telemetry type:
 | Option | Default value | Description |
 |--|--|--|
 | `Dashboard:ApplicationName` | `Aspire` | The application name to be displayed in the UI. This applies only when no resource service URL is specified. When a resource service exists, the service specifies the application name. |
+| `Dashboard:UI:DisableResourceGraph` | `null` | Disables the rendering of resource graphs on the dashboard. |
```

Comment:
```suggestion
| `Dashboard:UI:DisableResourceGraph` | `false` | Disables displaying the resource graph UI in the dashboard. |
```
null and false mean the same thing, so we might as well say false is the default.